.changeset/smooth-hornets-smash.md

```diff
@@ -0,0 +1,5 @@
+---
+'@hyperdx/browser': minor
```

Comment:
I think this change can be patch since we are still on 0.x.x

Reply:
updated